auto_backup/models/db_backup.py

```diff
@@ -18,7 +18,7 @@
 try:
     import pysftp
 except ImportError:
-    _logger.warning('Cannot import pysftp')
+    _logger.info('Cannot import pysftp')
```

Comment:
Shouldn't be better to use debug level as written at https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror ?